src/script/src/python/pyo3/vector_impl.rs

```diff
@@ -32,23 +32,23 @@ use crate::python::pyo3::utils::{pyo3_obj_try_to_typed_val, to_py_err};
 
 macro_rules! get_con_type {
     ($obj:ident, $($pyty:ident => $con_ty:ident),*$(,)?) => {
-    $(
-        if $obj.is_instance_of::<$pyty>()?{
-            Ok(ConcreteDataType::$con_ty())
+        $(
+            if $obj.is_instance_of::<$pyty>()?{
+                Ok(ConcreteDataType::$con_ty())
+            }
+        )else* else{
+            Err(PyValueError::new_err("Unsupported pyobject type: {obj:?}"))
         }
-    )else* else{
-        Err(PyValueError::new_err("Unsupported pyobject type: {obj:?}"))
-    }
     };
 }
 
 fn get_py_type(obj: &PyAny) -> PyResult<ConcreteDataType> {
     // Bool need to precede Int because `PyBool` is also a instance of `PyInt`
     get_con_type!(obj,
-        PyBool => boolean_datatype,
-        PyInt => int64_datatype,
-        PyFloat => float64_datatype,
-        PyString => string_datatype
+                  PyBool => boolean_datatype,
+                  PyInt => int64_datatype,
+                  PyFloat => float64_datatype,
+                  PyString => string_datatype
```

Comment:
style: Do these tabs look redundant?

Reply:
It's formatted by formatter. I don't know why.